deploy/charts/ray/values.yaml

```diff
@@ -83,3 +108,10 @@ operatorImage: rayproject/ray:latest
 # For a particular official release version of Ray, use `rayproject/ray:1.x.y`.
 # For a specific master commit, use the first 6 characters of the commit SHA, e.g. `rayproject/ray:050a07`.
 # The operator and Ray cluster can use different Ray versions, provided both versions are >= 1.2.0
+
+## imagePullSecret for the rey operator
+## Reference to one or more secrets to be used when pulling images
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+imagePullSecrets: []
```

Comment:
I think we should move operator settings into an operator subfield, while keeping compatibility with the current chart -- I can help out with this.  

I like the change of having the pull secrets be a per-pod setting.